.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```

Comment:
Would you mind explaining what this launch file is for?  Is it for testing??

Reply:
oops, I didnt mean to upload this one. This was actually generated from google chrome dev testing tools... I'll just remove it.